metricsaggregator/src/main/java/io/dockstore/metricsaggregator/client/cli/TerraMetricsSubmitter.java

```diff
@@ -199,7 +199,6 @@ private void executionMetricsPost(List<RunExecution> workflowExecutionsToSubmit,
         try {
             extendedGa4GhApi.executionMetricsPost(new ExecutionsRequestBody().runExecutions(workflowExecutionsToSubmit), Partner.TERRA.toString(), sourceUrlTrsInfo.trsId(),
                     sourceUrlTrsInfo.version(), description);
-            numberOfExecutionsSubmitted.addAndGet(workflowMetricRecords.size());
```

Comment:
Hmm, I would think this location would also only count it if there's no exception. I believe the problem is that we're logging the count of `workflowMetricRecords` which may not be the same as the workflow executions that are being submitted via `workflowExecutionsToSubmit`.

I think we should add the count of `workflowExecutionsToSubmit` instead of `workflowMetricRecords` because the former gets partitioned and contains the accurate count

Reply:
Oops, totally missed this and makes sense